### ISSUE TITLE
Basics description

### ISSUE THREAD

**Issue by rsc3** [linkml/schemasheets#125]
https://linkml.io/schemasheets/intro/basics/

The line starting with "the first two are slots." Shouldn't this say "the third is a CLASS"?

Also maybe qualify you mean the first two rows after the ">class" row, right?

**Comment by rsc3** [comment]
"The first two are slots, the third is a slot, and the fourth represents the slot id when used in the context of the Person class."

**Comment by cmungall** [comment]
Yes, the rows starting with `>` are *column descriptors*, the remaining ones are *element rows*, this is further up the page but it might be worth repeating, or emphasizing the difference between element rows; this is what it says at the moment:

"Looking at the first three element rows, we can see the first two represent fields (slots), in particular a field called id which is an identifier, and a field called description"

then

"The third element row represents a record (class) type - here, a Person"
